src/layouts/job-filters.tsx

```diff
@@ -41,27 +41,39 @@ const JobFilters = ({ searchParams }: { searchParams: JobQuerySchemaType }) => {
     },
   });
 
-  const formValues = form.watch();
+  async function handleFormSubmit(data: JobQuerySchemaType) {
+    await jobFilterQuery(
```

Comment:
please use the new setQueryParams hook , or if you're just resetting the page you can just reload the page without query params , something like  router.replace(pathname);